tests/lib/prepare.sh

```diff
@@ -85,7 +85,7 @@ disable_refreshes() {
 }
 
 setup_systemd_snapd_overrides() {
-    START_LIMIT_INTERVAL="StartLimitInterval=0"
+    START_LIMIT_INTERVAL="StartLimitInterval=100"
```

Comment:
Systemd seems to document `StartLimitIntervalSec=`. https://www.freedesktop.org/software/systemd/man/systemd.service.html - can you please check how this behaves across 16.04, 18.04 and 20.04 so that we have some understanding as to how reliable this setting is?

Can you cross reference systemd source code if both names are accepted.

Reply:
The commit that made the change from StartLimitInterval to StartLimitIntervalSec was for systemd 230 here: https://github.com/systemd/systemd/commit/f0367da7d1a61ad698a55d17b5c28ddce0dc265a. It is still backwards compatible in the [Unit] section like we are doing here, so this is safe to do so for all systemd versions we test with. If we were only running on bionic and later I would recommend changing this to be in the [Service] section and using the new name, but since we need to run on xenial as well with this, I think it's better to leave the most compatible version here.